README.md

```diff
@@ -152,6 +152,33 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
 }
 ```
 
+### RationaleCallbacks
+Implement this callback if you want to interact with the rationale dialog.
```

Comment:
Let's use the specific name, so something like "Implement the `EasyPermissions.RationaleCallbacks` if ..."